rfc/0003-policy-signing.md

```diff
@@ -0,0 +1,440 @@
+|              |                                  |
+| :----------- | :------------------------------- |
+| Feature Name | Policy Signing                   |
+| Start Date   | 21 Jan 2022                      |
+| Category     | Security                         |
+| RFC PR       | [#147](https://github.com/kubewarden/kubewarden-controller/pull/147)  |
+
+# Summary
+[summary]: #summary
+
+> Brief (one-paragraph) explanation of the feature.
+
+We want to provide a way to verify the integrity and authenticity of our policies, so that end-users can trust them.
+
+To accomplish that, we will provide ways to sign and verify policies.
+
+
+# Motivation
+[motivation]: #motivation
+
+> - Why are we doing this?
+> - What use cases does it support?
+> - What is the expected outcome?
+>
+> Describe the problem you are trying to solve, and its constraints, without coupling them too closely to the solution you have in mind. If this RFC is not accepted, the motivation can be used to develop alternative solutions.
+
+Our policies are binary blobs, that makes hard to ensure the policy we're about to run
+matches with the source code tagged inside of a git repository.
+
+This is what we want to achieve with this proposal:
+
+* policy authors: have a way to sign their policies, to attest their provenance.
```

Comment:
Hmm, policy authors are probably more interested in signing-off their source code commits.

Reply:
Policy authors are interested about this topic if they want to implement something like SLSA